src/components/common/LinearGauge.tsx

```diff
@@ -0,0 +1,28 @@
+import { LinearGaugeProps } from "../../@types/components/commonTypes";
+
+const LinearGauge = (props: LinearGaugeProps) => (
+  <div className={`flex ${props.containerClassName}`}>
+    <span
+      className={`flex items-center justify-center rounded-l-3xl w-[${Math.round(
+        (Number(props.propOne) /
+          (Number(props.propOne) + Number(props.propTwo))) *
+          100,
+      )}%] border-solid border-r-4 border-black-101 h-[${props.height}] ${
+        props.bgColorOne
+      }`}
+    >
+      {Number(props.propOne)}
```

Comment:
Please check the scenario in which props.propOne is undefined

Reply:
undefined cannot be passed.
https://github.com/raj-kumar-litmus/support-hub/blob/feat/donught-charts/src/%40types/components/commonTypes.tsx#L449-L450